src/ViewRenderer.php

```diff
@@ -75,7 +75,7 @@ public function __construct(
      */
     public function getViewPath(): string
     {
-        return $this->aliases->get($this->viewPath) . ($this->name ? '/' . $this->name : '');
+        return $this->aliases->get($this->viewPath);
```

Comment:
Why? Seems, it's return incorrect view path without name.

Reply:
It may confuses with scoped view.

Reply:
Reverted.